test/Microsoft.AspNetCore.Authentication.Test/JwtBearer/JwtBearerMiddlewareTests.cs

```diff
@@ -215,67 +183,37 @@ public class JwtBearerMiddlewareTests
         [Fact]
         public async Task RetrievingTokenFromAlternateLocation()
         {
-            var server = CreateServer(new JwtBearerOptions
+            var options = new JwtBearerOptions()
             {
                 AutomaticAuthenticate = true,
                 Events = new JwtBearerEvents()
                 {
-                    OnReceivingToken = context =>
+                    OnMessageReceived = context =>
                     {
                         context.Token = "CustomToken";
                         return Task.FromResult<object>(null);
-                    },
-                    OnReceivedToken = context =>
```

Comment:
Alright, I guess this change is the last step of the "completely revamp the bearer middleware to get rid of simple scenarios" plan initiated by @brentschmaltz last year? :smile: 

Without this event, it will be hard for people to simply associate an `AuthenticationTicket` with a token without relying on the default `ISecurityTokenValidator` logic.